hkm/static/hkm/js/hkm-v2.js

```diff
@@ -337,7 +343,50 @@ palikka
       });
     }
   }
+})
+.define('app.reset-password', ['jQuery', 'docReady'], function() {
+  $('#password-reset-email').on('submit', function(event) {
+    event.preventDefault();
 
+    $.post('/', {
+      action: 'password_reset',
+      csrfmiddlewaretoken: $('input[name="csrfmiddlewaretoken"]').val(),
+      email: $(this).find('input[name="email"]').val(),
+    }).done(function () {
+        $('#password-reset-form').addClass('hidden');
+        $('#password-reset-response-success').removeClass('hidden');
+    }).fail(function () {
+      $('#password-reset-form').addClass('hidden');
+      $('#password-reset-response-error').removeClass('hidden');
+    })
+  })
+
+  // Logic for opening the password reset modal automatically
+  const url = window.location.pathname;
+  const passwordChange = url.search('reset') > -1
+
+  if (passwordChange) {
+    $('#password-change').modal('toggle')
+  }
+
+  // POST password reset form data
+  $('#password-set-new').on('submit', function (event) {
+    event.preventDefault();
+
+    $.post(url, {
+      csrfmiddlewaretoken: $('input[name="csrfmiddlewaretoken"]').val(),
+      new_password1: $('#id_new_password1').val(),
+      new_password2: $('#id_new_password2').val()
```

Comment:
Minor comment, it's possible to get these values by using the on submit event. Up to you if you want to do so.